### PR TITLE
Feature/add availability tab

### LINKED ISSUE #4288: /event-type-detail / availalibility
Availaibility tab in event types should show the select preset's times. 

Image

### PR DESCRIPTION
**Pull request by emrysal**
## What does this PR do?

Closes #4288

image

### CODE REVIEW

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+  className?: string;
+  name: string;
+  value: number;
+  onBlur: () => void;
+  onChange: (value: AvailabilityOption | null) => void;
+}) => {
+  const query = trpc.useQuery(["viewer.availability.list"]);
+
+  return (
+    <QueryCell
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/AvailabilityTab.tsx:36]: NIT: To my understanding we are trying to move away from Query Cell? @zomars

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            const isAvailable = !!filterDays(index).length;
+            return (
+              <li key={day} className="my-6 flex border-transparent last:mb-2">
+                <span className={classNames("w-32 font-medium", !isAvailable && "text-gray-500")}>{day}</span>
+                {isLoading ? (
+                  <SkeletonText className="block" width="60" height="5" />
+                ) : isAvailable ? (
+                  <div className="space-y-3">
+                    {filterDays(index).map((dayRange, i) => (
+                      <div key={i} className="flex items-center">
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/AvailabilityTab.tsx:114]:
```suggestion
                      <div key={i} className="flex items-center leading-4">
```

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        />
+      </div>
+
+      <div className="space-y-4 rounded border p-8 py-6 pt-2">
+        <ol className="table border-collapse text-sm">
+          {weekdayNames(i18n.language, 1, "long").map((day, index) => {
+            const isAvailable = !!filterDays(index).length;
+            return (
+              <li key={day} className="my-6 flex border-transparent last:mb-2">
+                <span className={classNames("w-32 font-medium", !isAvailable && "text-gray-500")}>{day}</span>
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/AvailabilityTab.tsx:108]:
```suggestion
                <span className={classNames("w-32 font-medium", !isAvailable && "text-gray-500 opacity-50")}>{day}</span>
```

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+                    {filterDays(index).map((dayRange, i) => (
+                      <div key={i} className="flex items-center">
+                        <span className="w-28">{format(dayRange.startTime)}</span>
+                        <span className="">-</span>
+                        <div className="ml-6">{format(dayRange.endTime)}</div>
+                      </div>
+                    ))}
+                  </div>
+                ) : (
+                  <span className=" text-gray-500">{t("unavailable")}</span>
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/AvailabilityTab.tsx:122]:
```suggestion
                  <span className=" text-gray-500">{t("unavailable opacity-50")}</span>
```

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+                )}
+              </li>
+            );
+          })}
+        </ol>
+        <hr />
+        <div className="flex justify-between">
+          <span className="flex items-center text-sm">
```
Comment (sean-brydon) [inline apps/web/components/v2/eventtype/AvailabilityTab.tsx:130]:
```suggestion
          <span className="flex items-center text-sm text-gray-600">
```

**Review by sean-brydon — DISMISSED** [review]
LGTM - few NITS but I did it for you pretty much <3

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx (hunk 3 above)
Comment (emrysal) [reply apps/web/components/v2/eventtype/AvailabilityTab.tsx:108]: I was wondering why in Figma it didn't have the same colour

Comment (sean-brydon) [reply apps/web/components/v2/eventtype/AvailabilityTab.tsx]: Yeah a few places - especially disabled states are just modified with opacity

File: apps/web/components/v2/eventtype/AvailabilityTab.tsx (hunk 1 above)
Comment (PeerRich) [reply apps/web/components/v2/eventtype/AvailabilityTab.tsx:36]: is this a blocker? @zomars @emrysal @sean-brydon 

if not lets merge

Comment (sean-brydon) [reply apps/web/components/v2/eventtype/AvailabilityTab.tsx:36]: I don't believe so @PeerRich  - We still use it across the app. New issue should be created to remove query cell if we are planning to not use it anymore however

**Comment by emrysal** [comment]
@sean-brydon All addressed

**Comment by sean-brydon** [comment]
@baileypumfleet Looks good to me - design wise is perfect :) It works swell tarts a bonus

**Review by zlwaterfield — APPROVED** [review]
Looks awesome! 🥳

**Pull request merged**